addons/l10n_ae/tools.py

```diff
@@ -0,0 +1,26 @@
+JOURNALS_TO_CREATE = {"TA": {"name": "Tax Adjustments", "type": "general"},
+                      "IFRS": {"name": "IFRS 16", "type": "general"}}
+
+
+def create_journals(env):
```

Comment:
Normally there is a function when installing the chart template to prepare the journals.  _prepare_all_journals in account/models/chart_template.py.  You might be able to inherit those.  

Are this journals always needed?

Reply:
Yeah these journals are standard here. I will look into the function you suggested and fix the code.
Thank you @jco-odoo

Reply:
I am now creating the journals using the _prepare_all_journals method as you have suggested. I am still using a post-hook to link the I`FRS` accounts to the newly created `IFRS 16` journal. Is this approach okay or do you think there's a better way to do it?